data_access_service/core/routes.py

```diff
@@ -214,96 +227,105 @@ def _remove_file_if_exists(file_path):
     return response
 
 
-@restapi.route("/health", methods=["GET"])
-def health_check() -> Response:
-    return Response("healthy", mimetype="application/json")
+class HealthCheckResponse(BaseModel):
+    status: str
 
 
-@restapi.route("/metadata", methods=["GET"])
-@restapi.route("/metadata/<string:uuid>", methods=["GET"])
-def get_mapped_metadata(uuid=None):
+@router.get("/health", response_model=HealthCheckResponse)
+async def health_check():
+    """
+    Health check endpoint.
+    """
+    return HealthCheckResponse(status="healthy")
```

Comment:
Seems need a bit more check before report healthy for example when the app is init from s3 we should not report healthy. The status is either "UP" or "DOWN" or "STARTING", if it is starting and reading s3, please report starting